apps/core-app/src/legos/fields.ts

```diff
@@ -39,4 +39,46 @@ export const FIELD: Record<string, CustomFieldLego> = {
     type: 'toWeiInput',
     label: 'Should Overwrite',
   },
+  PROPOSAL_EXPIRY: {
+    id: 'checkRender',
+    type: 'checkRender',
+    gateLabel: 'Add Expiration Date',
+    customFields: CustomFields,
+    components: [
+      {
```

Comment:
Wondering why all three fields couldn't be wrapped in one component. Not a big deal for now.

Reply:
I'm guess it would be nice to have a highlighted text generic, nvm.

Reply:
I refactored the code to include everything within a single custom component. I'd like to sync up with you sometime on how to make the HighlightedText more generic

Reply:
let me know what you think about how I refactored the component. If it's okay I'd squash the commits before merging otherwise I can switch back to the previous version